app/assets/javascripts/oxalis/view/layouting/default_layout_configs.js

```diff
@@ -146,8 +146,8 @@ export const getCurrentDefaultLayoutConfig = () => {
   };
 };
 
-export function determineLayout(controlMode: ControlMode, viewMode: Mode): Layout {
-  if (controlMode === ControlModeEnum.VIEW) {
+export function determineLayout(viewMode: Mode): Layout {
+  if (Constants.MODE_PLANE_TRACING === viewMode) {
```

Comment:
I don't think this is the correct change. Instead you should keep the `if (controlMode === ControlModeEnum.VIEW) {` and add an if/else like in line 159:
```
if (isArbitraryMode) {
  return "ArbitraryLayoutView";
} else {
  return "OrthoLayoutView"
}
```
`ArbitraryLayoutView` then needs to be added to the `mapLayoutKeysToLanguage` object.
This is because we're essentially adding a new layout, abitrary tracing in view-only mode.
Otherwise, the wrong tabs are shown when skeletontracing (try it out with the deployed version).

Reply:
I had to add a whole new layout called  `ArbitraryLayoutView` for this. I hope I did find every file I had to adjust for this